pkg/agent/openflow/pipeline.go

```diff
@@ -1466,6 +1466,7 @@ func (c *client) conjunctionActionFlow(conjunctionID uint32, tableID binding.Tab
 		if enableLogging {
 			return c.pipeline[tableID].BuildFlow(ofPriority).MatchProtocol(proto).
 				MatchConjID(conjunctionID).
+				Action().Meter(1).
```

Comment:
you should be able to use your constant here

Reply:
Done.